lib/chemotion/inventory_service.rb

```diff
@@ -0,0 +1,212 @@
+# frozen_string_literal: true
+
+module Chemotion::InventoryService
```

Comment:
Please refactor this into a proper class, so you can reduce redundant code (like for example always the same http options) into private methods